settings.gradle.kts

```
@@ -80,6 +80,7 @@ include(":misk-rate-limiting-bucket4j-dynamodb-v1")
 include(":misk-rate-limiting-bucket4j-mysql")
 include(":misk-rate-limiting-bucket4j-redis")
 include(":misk-redis")
+include("misk-schema-migrator-gradle-plugin")
```

```suggestion
include(":misk-schema-migrator-gradle-plugin")
```

Is the missing `:` deliberate?

---

Just what IntelliJ generated I think